packages/core/src/utils.ts

```diff
@@ -593,7 +593,7 @@ export function toTransitionConfigArray<TContext, TEvent extends EventObject>(
     if (
       typeof transitionLike === 'undefined' ||
       typeof transitionLike === 'string' ||
-      isMachine(transitionLike)
+      isMachineNode(transitionLike)
```

Comment:
is this check `necessary`? a machine itself cannot be a target of a transition

Reply:
This is meant in terms of `isStateNode(...)` but I don't have that function yet.